spec/models/crop_spec.rb

```diff
@@ -328,6 +328,8 @@
   context "search" do
     before :each do
       @mushroom = FactoryGirl.create(:crop, :name => 'mushroom')
+      @mushroom.__elasticsearch__.index_document
+      sleep 1
```

Comment:
This sleep is here to manage the asynchronicity of the indexing, right? Might be better to handle this without `sleep`, because this can sometimes cause spuriously failing tests. I'm not sure if this is possible without doing a little research. If it turns out to be a big hassle then we can probably live with this since indexing one record should be super fast.